### PR TITLE
Update images used by kaniko-executor and remove minio

### PR DESCRIPTION
**Pull request by m00g3n**
**Description**

Update images used by kaniko-executor to use SHA256 instead tags.

Changes proposed in this pull request:

- update golang image
- update buster-slim
- remove minio

**Related issue(s)**

### CODE REVIEW

File: kaniko-executer/Dockerfile [hunk 1]
```diff
@@ -12,9 +12,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Builds the static Go image to execute in a Kubernetes job
+# Builds the static Go image (1.18.1-alpine3.15) to execute in a Kubernetes job
 
-FROM eu.gcr.io/kyma-project/external/golang:1.18.1-alpine3.15 as builder
+# image builder base on golang:1.19.0-alpine3.16
+FROM golang@sha256:78767a599f5758e3fd60fd065de9dc40aec1ad3b86ded2bddc92640a465cb515 as builder
```
Comment (dbadura) [inline kaniko-executer/Dockerfile:18]: I found different sha:  `sha256:f94174c5262af3d8446833277aa27af400fd1a880277d43ec436df06ef3bb8ab`

Comment (dbadura) [inline kaniko-executer/Dockerfile:18]: Why do we switch from `eu.gcr.io` to docker hub?
I can't find this sha, this version was release 12 days ago:
https://hub.docker.com/layers/golang/library/golang/1.19.0-alpine3.16/images/sha256-70df3b8f9f099da7f60f0b32480015165e3d0b51bfacf9e255b59f3dd6bd2828?context=explore

Comment (m00g3n) [reply kaniko-executer/Dockerfile:18]: the lack of `eu.gcr.io/kyma-project/external/golang` registry it a copy/paste error; will fix that

Comment (m00g3n) [reply kaniko-executer/Dockerfile:18]: thx

**Review by pPrecel — DISMISSED** [review]

**Review by grego952 — DISMISSED** [review]

File: kaniko-executer/Dockerfile [hunk 2]
```diff
@@ -12,9 +12,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Builds the static Go image to execute in a Kubernetes job
+# Builds the static Go image (1.18.1-alpine3.15) to execute in a Kubernetes job
```
Comment (dbadura) [inline kaniko-executer/Dockerfile:15]: Do we build go 1.18 or 1.19?

Comment (m00g3n) [reply kaniko-executer/Dockerfile]: comment updated, thx

**Review by dbadura — DISMISSED** [review]

**Review by grego952 — DISMISSED** [review]

**Review by moelsayed — DISMISSED** [review]

**Review by pPrecel — DISMISSED** [review]
/lgtm
/approve
/meow

**Review by pPrecel — APPROVED** [review]

**Pull request merged**